fn/routes.go

```diff
@@ -278,7 +278,8 @@ func envAsHeader(req *http.Request, selectedEnv []string) {
 }
 
 func (a *routesCmd) create(c *cli.Context) error {
-	if c.Args().Get(0) == "" || c.Args().Get(1) == "" || c.Args().Get(2) == "" {
+	// todo: @pedro , why aren't you just checking the length here?
+	if c.Args().Get(0) == "" || c.Args().Get(1) == "" {
```

Comment:
@treeder I haven't changed so many things here. @ccirello checked like that, and I just increase one more argument validation.
But just validate the length is a better solution.